Tests/SwiftDocCTests/Infrastructure/DocumentationContext/DocumentationContextTests.swift

```diff
@@ -644,15 +644,25 @@ class DocumentationContextTests: XCTestCase {
     
     func testDetectsReferenceCollision() throws {
         let (_, context) = try testBundleAndContext(named: "TestBundleWithDupe")
-        
-        XCTAssert(
-            context.problems.contains { problem in
-                problem.diagnostic.identifier == "org.swift.docc.DuplicateReference"
-                    && problem.diagnostic.localizedSummary == "Redeclaration of 'TestTutorial.tutorial'; this file will be skipped"
-            }
-        )
+
+        let problemWithDuplicate = context.problems.filter{ $0.diagnostic.identifier == "org.swift.docc.DuplicateReference" }
+
+        XCTAssert(problemWithDuplicate.count == 1)
+
+        XCTAssert(problemWithDuplicate[0].diagnostic.localizedSummary == "Redeclaration of 'TestTutorial.tutorial'; this file will be skipped")
+
     }
     
+    func testDetectsMultipleMDfilesWithSameName() throws {
+        let (_, context) = try testBundleAndContext(named: "TestBundleWithDupMD")
+
+        let problemWithDuplicateReference = context.problems.filter { $0.diagnostic.identifier == "org.swift.docc.DuplicateReference" }
+
+        XCTAssert(problemWithDuplicateReference.count==2)
+
+        XCTAssert(problemWithDuplicateReference[0].diagnostic.localizedDescription == "/Users/sradmin/Library/Developer/Xcode/DerivedData/swift-docc-dqymopzdtzbqmnbkvrtkjxsnnyyt/Build/Products/Debug/SwiftDocCTests.xctest/Contents/Resources/SwiftDocC_SwiftDocCTests.bundle/Contents/Resources/Test Bundles/TestBundleWithDupMD.docc/documentation1/overview.md: warning: Redeclaration of \'overview.md\'; this file will be skipped\nThis content was already declared at \'file:///Users/sradmin/Library/Developer/Xcode/DerivedData/swift-docc-dqymopzdtzbqmnbkvrtkjxsnnyyt/Build/Products/Debug/SwiftDocCTests.xctest/Contents/Resources/SwiftDocC_SwiftDocCTests.bundle/Contents/Resources/Test%20Bundles/TestBundleWithDupMD.docc/documentation/overview.md\'")
```

Comment:
This is likely going to fail in CI. We should try to avoid using paths specific to a certain user account in test cases.

Reply:
yeah, i updated with check on localizedSummary